rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md

```diff
@@ -44,15 +41,19 @@ If you believe the issues being raised are not important, please check the secti
 ### Modify Visual Studio Analysis Ruleset
 
 The goal is to develop a shared ruleset across projects. (Currently this is just the default settings). This will ensure the same standard and quality of code is maintained across all of the company's projects.
-Any project specific rules should be documented in "\_Instructions-CodeHealth.docx" which is to be kept in the solution.
-Please also copy the current version number of this rule into the "\_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
+Any project specific rules should be documented in "_Instructions-CodeHealth.docx" which is to be kept in the solution.
+Please also copy the current version number of this rule into the "_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
+
+Starting in Visual Studio 2019, you can configure the severity of analyzer rules in an EditorConfig file.
+
+EditorConfig file can belong to the Solution for the Solution-wide rules and to the Project for the Project-specific rules.
+
+If you do not have the EditorConfig file yet, right click on either the project or solution node and go to Add | New Item. Next, select "editorconfig File (.Net)" and click Add.
 
-![Figure: Steps to open Visual Studio Analyser rules customisation page](VS-ModifyRules.png)
-\
-Steps to open Analyser customisation page:
+![Figure: Add EditorConfig File](add-editor-config.png)
 
-Right Click project &gt; Properties &gt; Code Analysis &gt; Open
+To edit the ruleset, open your .editorconfig file.
 
-![Figure: How to customize rules. By either enabling / disabling rules or packages. Or by modifying the rule severity level](VS-ModifyRules2.png)
+In the Analyzers tab you can change the severity of the rules, for example disable the warning or convert it into an error.
 
-![Figure: How to apply custom ruleset to all projects in a solution](VS-ModifyRules3.png)
+![Figure: Configure your Rules in the in the EditorConfig | Analyzers](configure-rules.png)
```

Comment:
We have a specific rule for .editorconfig.
https://www.ssw.com.au/rules/consistent-code-style/

I would remove this section and replace it with a link to that rule

Reply:
Done - I mention that we can use EditorConfig and then just forward them to the [rule](https://www.ssw.com.au/rules/consistent-code-style/)